### ISSUE TITLE
Obsidian uri link error

### ISSUE THREAD

**Issue by 1qzhworld** [argenos/zotero-mdnotes#172]
I am using the Zotero 6.0.5-beta.2+c510d9146 and the latest versions of zotfile, better bibtex and mdnotes.
(Additionaly, mdnotes v.0.2.0-alpha5 - Zotero 6 compatibility, though in Zotero it still shows "mdnotes for Zoter 0.1.3"?)

I have set the configurations in the advanced preference as mentioned in [Hidden preferences](https://argentinaos.com/zotero-mdnotes/docs/next/advanced/hidden-prefs/). However, I can not open the generated URI.

I am have try to figure out what happened. And I thought it may related to the Note in [Hidden preferences](https://argentinaos.com/zotero-mdnotes/docs/next/advanced/hidden-prefs/), where I should use "Shortest path". But I do not find where to set that.

In short, I get uri file with "obsidian://open?vault=test&file=D:/test/open_this". And this open Obsidian with warning that file "D:/test/open_this" not exist.
The correct URI should be "obsidian://open?vault=test&file=open_this" or ""obsidian://open?vault=test&path=D:/test/open_this"

Is there any suggestion?

(I think this should be irrelated to the Obsidian setting, though I also set the "Files \& Links->New link format->Shortest path when possible")

**Comment by rxy0424** [comment]
PR #108 and #166 fix the problem, which have been merged into the develop branch.

**Comment by 1qzhworld** [comment]
> PR #108 and #166 fix the problem, which have been merged into the develop branch.

Thanks for you reminder. It works!
